internal/database/own_signal_configurations_test.go

```diff
@@ -0,0 +1,64 @@
+package database
+
+import (
+	"context"
+	"testing"
+
+	"github.com/hexops/autogold"
```

Comment:
Do you want to use v2? I think there is a [linter](https://sourcegraph.com/github.com/sourcegraph/sourcegraph@03da2be83dca2f60e973e893a69d8053658d7129/-/blob/dev/linters/depguard/depguard.go?L23:3) checking for it.

```suggestion
	"github.com/hexops/autogold/v2"
```

Reply:
nice, had no idea there is a new version!